run_tests.sh

```diff
@@ -1,12 +1,14 @@
 #!/bin/sh
 
 export PYTHONPATH=.
-echo "Running test_partner.py ----------------------------------------------\n\n"
+printf "Running test_partner.py ----------------------------------------------\n\n"
```

Comment:
why is printf better than echo?

Reply:
echo can behave different on different systems. There can be different interpretations, especially with `\n`, while printf can handle that.